### PR TITLE
Redesign contract functions overview

### PR DESCRIPTION
**Pull request by shift4id**

### CODE REVIEW

File: contract-ui/tabs/explorer/page.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  contractAddress,
+}) => {
+  const { contract, isSuccess, isError } = useContract(contractAddress);
+
+  const functions = useContractFunctions(contract?.abi as Abi);
+  if (!contractAddress) {
+    return <div>No contract address provided</div>;
+  }
+
+  if ((!contract?.abi && isSuccess) || isError) {
```
Comment (nachoiacovino) [inline contract-ui/tabs/explorer/page.tsx:22]: do we need to do this here? right now we only do it on overview

**Review by jnsdls — APPROVED** [review]
LGTM

File: contract-ui/tabs/explorer/page.tsx (hunk 1 above)
Comment (jnsdls) [reply contract-ui/tabs/explorer/page.tsx:22]: yeah only overview is fine for this

**Review by nachoiacovino — APPROVED** [review]
love the new design, looks great on its own tab

**Comment by nachoiacovino** [comment]
one improvement, I will make only the functions scroll, not the whole thing, would look better on the releases page 
image

**Pull request merged**